app/lib/message_list/message_renderer.dart

```diff
@@ -11,12 +11,21 @@ import 'package:prototype/core/api/markdown.dart';
 Widget buildBlockElement(BlockElement block, bool isSender) {
   return switch (block) {
     BlockElement_Paragraph(:final field0) => Text.rich(
-      TextSpan(children: field0.map(buildInlineElement).toList()),
+      TextSpan(
+        children:
+            field0.map((child) => buildInlineElement(child, isSender)).toList(),
+        style: TextStyle(color: isSender ? Colors.white : Colors.black),
+      ),
     ),
     BlockElement_Heading(:final field0) => Text.rich(
       TextSpan(
-        children: field0.map(buildInlineElement).toList(),
-        style: const TextStyle(fontSize: 20, fontWeight: FontWeight.bold),
+        children:
+            field0.map((child) => buildInlineElement(child, isSender)).toList(),
+        style: TextStyle(
+          fontSize: 20,
+          fontWeight: FontWeight.bold,
```

Comment:
Font weight has no effect with variable fonts, you have to use font variations instead, as defined [here](https://github.com/phnx-im/infra/blob/bb62f156c474f5c0c0ee5405c8912fcf2e08580d/app/lib/theme/styles.dart#L35).

Reply:
Fixed in https://github.com/phnx-im/infra/pull/529